custom/ilsgateway/views.py

```diff
@@ -345,6 +346,31 @@ def get_redirect_url(self, domain, *args, **kwargs):
         return url
 
 
+class ZiplineOrdersView(View):
+
+    @method_decorator(login_and_domain_required)
```

Comment:
I don't really like the introduction of this view because it gives anyone in the domain access to a list of order ids, and it's hard to properly do authorization on this view because it requires knowing who has access to the reports that use the filter.

I think it would be better to just use a simple text input as the order id filter in the reports (and only allow filtering on one order), and remove this view.